bugbot/rules/file_crash_bug.py

```diff
@@ -63,24 +98,18 @@ def get_bugs(self, date):
             # TODO: Handle cases where the regressor is a security bug. In such
             # cases, we may want to file the bug as security bug.
 
-            flags = None
-            if signature.regressed_by:
-                # TODO: check user activity and if the ni? is open
-                flags = [
-                    {
-                        "name": "needinfo",
-                        "requestee": signature.regressed_by_author["name"],
-                        "status": "?",
-                        "new": "true",
-                    }
-                ]
+            # Weather we should needinfo the regression author.
```

Comment:
```suggestion
            # Whether we should needinfo the regression author.
```

Reply:
Done in 65ae5c269658bfdb2a9c28800c43a6a03fb5c425.